code/codegen/src/main/kotlin/com/android/gradle/replicator/codegen/gradlegen/BuildFileGenerator.kt

```diff
@@ -0,0 +1,10 @@
+package com.android.gradle.replicator.codegen.gradlegen
+
+import com.android.gradle.replicator.codegen.PrettyPrintStream
+
+/**
+ * Generates a build file for a build system.
+ */
+interface BuildFileGenerator {
```

Comment:
I don't think that's necessary. Let's keep this code only.

Reply:
I use it in test. I guess I can move it there or rename it TestBuildFileGenerator ?

Reply:
Should probably be in the test code rather than the generator code then, but it's fine for now.